src/components/common/BesideNavbar/BesideNavbar.css

```diff
@@ -0,0 +1,76 @@
+.BesideNavbar{
+    width:35vw;
+    display: flex;
+    background-color: var(--black-hl-p5);
+    display: flex;
+    padding: 10px;
+    position: sticky;
+    top: 0;
+}
+
+
+.beside-navbar-container{
+    display: flex;
+    justify-content: flex-start;
```

Comment:
this is not needed, as this is the default property

Reply:
oh got it